src/clj/puppetlabs/services/config/puppet_server_config_service.clj

```diff
@@ -27,11 +27,11 @@
           (ks/pprint-to-string puppet-config))
         (core/init-webserver! override-webserver-settings!
                               (get-in tk-config
-                                      [:webserver :puppet-server]
+                                      [:webserver :puppetserver]
```

Comment:
I think it's unlikely this would affect OSS users.  The default 'webserver.conf' file that we ship for OSS has a single nameless server - https://github.com/puppetlabs/puppetserver/blob/puppet-server-2.3.1/ezbake/config/conf.d/webserver.conf.  I suppose some users could have observed that we have special logic in Puppet Server around the "puppet-server" name and taken advantage of that for doing multi-server configurations, but that doesn't seem too likely.

The "puppet-server" name is used in PE, as you mentioned, though - https://github.com/puppetlabs/pe-puppetserver/blob/31613598bec1aad3b7c7c5b7e6b4355ccd7f18c5/config/conf.d/webserver.conf#L2 - and probably would require PE module changes to go along with this.  That wouldn't necessarily make this untenable but where it is possible that users may have made customizations to the webserver.conf in PE outside of what the PE module manages - since I don't think we overwrite that file completely from the PE module anymore - it does seem like we could break some PE users.

For that reason, I'm inclined to agree with you and suggest that we preserve "puppet-server" here instead.  I'll make that change in a new commit.  /CC @cprice404 for awareness on this since I'd just pulled over his change for this originally and he may have a different opinion about it.